assembly/ScaleString.ts

```diff
@@ -0,0 +1,42 @@
+import { Bytes } from "./utils/Bytes"
+
+export class ScaleString extends Array<u8> {
+    constructor(input: string) {
+        const inputBuffer: ArrayBuffer = String.UTF8.encode(input);
```

Comment:
I think that it will be useful to have the `input` as a `readonly` property or at least have `getValue` or `getString` function in order to return the `string` representation of the `ScaleString`. I think we need this functionality because when we do `ScaleString.fromU8a` we are returning `ScaleString`, but the user won't be able to actually get the `string` value out of it, unless we add the `input` as a property in the `class`.
```TypeScript
const scaleStr = ScaleString.from8Ua([0x18]);
// I would like to be able to do scaleStr.value or scaleStr.getValue() and receive the String representation.
```